### DISCUSSION TITLE
how to change message components style

### DISCUSSION THREAD

**Discussion by tureo** [bwmarrin/discordgo#1266] [Q&A] [answered]
how can i change message components style when user click one of message components，
i wont to change （original message）： 
![image](https://user-images.githubusercontent.com/29032796/197758746-4c99ca38-52fc-42b1-b1e0-5f51bf9fb27a.png)

but got （a new message）：
![image](https://user-images.githubusercontent.com/29032796/197758978-13e4b232-aac0-4f19-af0b-f3d8b720c570.png)

**Comment by Nv7-GitHub** [comment, marked as answer]
make sure to use `InteractionResponseUpdateMessage` for the `Type`

**Reply by tureo** [reply]
thanks, You're right, now original message components style changed when i use InteractionResponseUpdateMessage for the Type
